src/Order.js

```diff
@@ -0,0 +1,28 @@
+import React, {useState} from 'react';
+import RestaurantButtons from './RestaurantButtons';
+
+const Order = ({orderType}) => {
+
+    const [order, setOrder] = useState(0);
+
+    const clickHandler = () => {
+        setOrder(order + 1);
+    }
+
+    const clickRemover = () => {
```

Comment:
We might be able to think of a better names for these `clickHandler` and `clickRemover` 🤔 

Maybe we could relate there name to the state they update? Something like, `addOrder`?